packages/occ-worker/src/occapi/index.ts

```diff
@@ -66,6 +66,11 @@ export const Fillet = operatorCache<IFillet>('Part::Fillet', _Fillet);
 export const ObjectFile = operatorCache<{
   content: string;
   type: IAny['Type'];
+  placement: {
```

Comment:
```suggestion
  placement?: {
```

Reply:
The generic type parameter here takes the same type of the first argument of `_loadObjectFile`

Reply:
> The generic type parameter here takes the same type of the first argument of `_loadObjectFile`

It is of same type as type in `_loadObjectFile`, do we want to change?